packages/components/src/toolbar/toolbar-button-container.js

```diff
@@ -0,0 +1,9 @@
+const ToolbarButtonContainer = ( props ) => (
+	<div
+		key={ props.keyProp }
```

Comment:
You don't need to pass it down, it is used only by React for its internal optimizations. See related React doc: https://reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys.